src/orderbook.js

```diff
@@ -164,7 +201,15 @@ OrderBook.EVENTS = [
   'offer_changed', 'offer_funds_changed'
 ];
 
-OrderBook.DEFAULT_TRANSFER_RATE = 1000000000;
+OrderBook.DEFAULT_TRANSFER_RATE = Amount.from_json(1000000000);
+
+OrderBook.NOTIFY_THROTTLE_TIME = 4000;
```

Comment:
Still don't see the use for debounce here. If I will set wait and maxWait to same value, it effectively be `_throttle`, and if wait will be smaller than maxWait, then it will not serve its main purpose - to make sure not to overload users cpu by not allowing to run autobridge compute to often.